7.1.цикл_for/7.1.9.py

```diff
@@ -0,0 +1,3 @@
+num = int(input())
+for item in range(num, 0, - 1):
+    print(item * "*")
```

Comment:
Дружище, это очень сложно читать. Так лучше:
```suggestion
for i in range(number):
    print("*" * item)
    number -= 1
```